kore/src/Kore/Log/KoreLogOptions.hs

```diff
@@ -106,6 +111,7 @@ instance Default KoreLogOptions where
             , logEntries = mempty
             , debugSolverOptions = def @DebugSolverOptions
             , exeName = ExeName mempty
+            , startTime = fromNanoSecs 0
```

Comment:
I don't really like this as a default because it doesn't make any sense to have time start at the beginning of the Unix epoch. Can we remove the `Default` instance?

Reply:
Do you mean removing the `Default` instance for `KoreLogOptions`?

Reply:
I spent some time trying to remove the `Default` instance and it doesn't seem straightforward. I'll try again tomorrow